freshdesk/v2/api.py

```diff
@@ -30,9 +30,14 @@ class TicketAPI(object):
     def __init__(self, api):
         self._api = api
 
-    def get_ticket(self, ticket_id):
-        """Fetches the ticket for the given ticket ID"""
-        url = "tickets/%d" % ticket_id
+    def get_ticket(self, ticket_id, *include):
+        """
+            Fetches the ticket for the given ticket ID
+            You can pass strings for the include parameter and they'll be included as include params to the request
+            ex: get_ticket(some_id, "stats", "conversations", "requester", "company") will result in the following request:
+            tickets/[some_id]?include=stats,conversations,requester,company
+        """
+        url = "tickets/%d%s" % (ticket_id, "?include=%s" % ",".join(include) if include else "")
```

Comment:
If there are no `include` args passed, then the URL will be constructed as `tickets/1?include=` - even if this works OK I would prefer to not include the query string if it's going to be empty.

Reply:
mmm the if-else construct ensures that ?include= is not added to the string if  there are no *include args ... I tested this on a Python console and some random strings...

Reply:
```
>>> def fun(id,*include):
           return "tickets/%d%s" % (id, "?include=%s" % ",".join(include) if include else "")
>>> fun(1, "stats", "hello")
'tickets/1?include=stats,hello'
>>> fun(1, "stats")
'tickets/1?include=stats'
>>> fun(1)
'tickets/1'
```

Reply:
You are right, I must have misread it originally.